slick/src/test/java/akka/stream/alpakka/slick/javadsl/DocSnippetFlow.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package example;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import akka.Done;
+import akka.actor.ActorSystem;
+import akka.stream.ActorMaterializer;
+import akka.stream.Materializer;
+import akka.stream.javadsl.Sink;
+import akka.stream.javadsl.Source;
+
+import akka.stream.alpakka.slick.javadsl.*;
+
+//#flow-example
```

Comment:
might be good to include some of the most important imports also

Reply:
Done